--- _deps/mdbx-src/mdbx.c.orig  2026-03-26 22:12:17.007272000 +0300
+++ _deps/mdbx-src/mdbx.c       2026-03-26 22:08:11.626424000 +0300
@@ -68,7 +68,7 @@
 #endif /* _NO_CRT_STDIO_INLINE */

 #elif !defined(_POSIX_C_SOURCE)
-#define _POSIX_C_SOURCE 200809L
+//#define _POSIX_C_SOURCE 200809L
 #endif /* Windows */

 #ifdef __cplusplus
@@ -28756,7 +28756,7 @@
     for (ior_item_t *item = ior->pool; item <= ior->last;) {
       if (!HasOverlappedIoCompleted(&item->ov)) {
         assert(ior->overlapped_fd);
-        CancelIoEx(ior->overlapped_fd, &item->ov);
+        CancelIo(ior->overlapped_fd);
       }
       if (item->ov.hEvent && item->ov.hEvent != ior)
         ior_put_event(ior, item->ov.hEvent);
