diff --git a/src/osal.c b/src/osal.c
index 5a0ae633..ec56591c 100644
--- a/src/osal.c
+++ b/src/osal.c
@@ -1090,7 +1090,7 @@ MDBX_INTERNAL_FUNC void osal_ioring_reset(osal_ioring_t *ior) {
     for (ior_item_t *item = ior->pool; item <= ior->last;) {
       if (!HasOverlappedIoCompleted(&item->ov)) {
         assert(ior->overlapped_fd);
-        CancelIoEx(ior->overlapped_fd, &item->ov);
+        CancelIo(ior->overlapped_fd);
       }
       if (item->ov.hEvent && item->ov.hEvent != ior)
         ior_put_event(ior, item->ov.hEvent);
